### PR TITLE
clean(ZMS-2375): made title of websites distinctive

### PR DESCRIPTION
**Pull request by msfb3**
**Description**

Short description or comments

**Reference**

Issues #XXX

## Summary by CodeRabbit

- **New Features**
	- Updated webpage titles for various templates to simplify branding, changing from "Zeitmanagementsystem" to "ZMS" or "ZMS Statistik".
  
- **Bug Fixes**
	- Adjusted the expected output in tests to reflect the new title "Kategorie auswählen" in the rendering process.

- **Chores**
	- Maintained overall layout and structure across templates without additional modifications.

### CODE REVIEW

File: zmsadmin/src/Zmsadmin/Application.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ class Application extends \BO\Slim\Application
 
     const DEBUG = false;
 
-    const TWIG_CACHE = '/cache/';
+    const TWIG_CACHE = false;
```
Comment (ThomasAFink) [inline zmsadmin/src/Zmsadmin/Application.php:30]: Twig Cache muss wieder rein

File: zmsstatistic/src/Zmsstatistic/Application.php [hunk 2]
```diff
@@ -23,7 +23,7 @@ class Application extends \BO\Slim\Application
 
     const DEBUG = false;
 
-    const TWIG_CACHE = '/cache/';
+    const TWIG_CACHE = false;
```
Comment (ThomasAFink) [inline zmsstatistic/src/Zmsstatistic/Application.php:26]: Hier auch Twig cache wieder setzten

File: zmsadmin/src/Zmsadmin/Application.php (hunk 1 above)
Comment (msfb3) [reply zmsadmin/src/Zmsadmin/Application.php]: Twig cache wurde gesetzt.

File: zmsstatistic/src/Zmsstatistic/Application.php (hunk 2 above)
Comment (msfb3) [reply zmsstatistic/src/Zmsstatistic/Application.php]: Twig cache wurde gesetzt.

**Review by manjencic — APPROVED** [review]

**Review by ThomasAFink — APPROVED** [review]

File: zmscalldisplay/package-lock.json [hunk 3]
```diff
@@ -183,45 +183,6 @@
         "@lezer/common": "^1.0.0"
       }
     },
-    "node_modules/@lmdb/lmdb-darwin-arm64": {
```
Comment (ThomasAFink) [inline zmscalldisplay/package-lock.json:186]: Package no longer exists..

**Review by ThomasAFink — CHANGES_REQUESTED** [review]

**Review by ThomasAFink — APPROVED** [review]

**Pull request merged**